planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -1031,10 +1031,7 @@ void planning_scene_monitor::PlanningSceneMonitor::onStateUpdate(const sensor_ms
       update = true;
     }
   }
-
-  // run the state update with state_pending_mutex_ unlocked
-  if (update)
-    updateSceneWithCurrentState();
+   updateSceneWithCurrentState(update);
```

Comment:
seems like removing this if statement might overload the planning scene with too many updates, depending on the frequency of joint_state broadcasts. Currently the state update functionality appears to be throttled to every 0.1 seconds, as hard coded in ``dt_state_update_``, right?